### DISCUSSION TITLE
PlantUML not rendering

### DISCUSSION THREAD

**Discussion by gtozzi** [requarks/wiki#7229] [Error / Bug Report]
Hi! Thank you for this great software!

I am trying to use the nice builting PlantUML support.

When adding a couple of PlantUML diagrams in the page, the diagram get rendered correctly in preview mode, while in the standard page view mode, the diagram gets rendered as a black code block containing its source code.

I suspect there is a bug in the rendering pipeline order.

- PlantUML is enabled in the pipeline
- I've cleared all the pages cache
- WikiJS is updated to the latest version
- It does make difference if using svg or png format
- The page is in Markdown format
- Trying in a different page doesn't make any difference
- Trying on a brand new page doesn't make any difference

**Comment by NGPixel** [comment]
Try to disable "Sanitize HTML" under Admin > Rendering > HTML > Security. Then re-save your page.

**Reply by gtozzi** [reply]
Thanks! Tried, didn't work.

Also tried flushing the cache and completely disabling the "Security" HTML renderer module.
